chaindexing-postgres/src/drizzle/schema.ts

```diff
@@ -1,15 +1,33 @@
-import { integer, pgTable, serial, text, uuid, json, date, boolean } from 'drizzle-orm/pg-core';
+import {
+  boolean,
+  date,
+  integer,
+  json,
+  pgTable,
+  serial,
+  text,
+  uniqueIndex,
+  uuid
+} from 'drizzle-orm/pg-core';
 
-export const chaindexing_contract_addresses = pgTable('chaindexing_contract_addresses', {
-  id: serial('id').primaryKey(),
-  chainId: integer('chain_id'),
-  lastIngestedBlockNumber: integer('last_ingested_block_number'),
-  startBlockNumber: integer('start_block_number'),
-  address: text('address'),
-  contractName: text('contract_name')
-});
+export const chaindexingContractAddressesSchema = pgTable(
+  'chaindexing_contract_addresses',
+  {
+    id: serial('id').primaryKey(),
+    chainId: integer('chain_id'),
+    lastIngestedBlockNumber: integer('last_ingested_block_number'),
+    startBlockNumber: integer('start_block_number'),
```

Comment:
Sorry, forgot to mention. Some of these fields are stale.

```suggestion
    startBlockNumber: integer('start_block_number'),
    nextBlockNumberToIngestFrom: integer('next_block_number_to_ingest_from'),
    nextBlockNumberToHandleFrom: integer('next_block_number_to_handler_from'),
```